cms/static/js/spec/views/login_studio_spec.js

```diff
@@ -0,0 +1,30 @@
+define(['jquery', 'js/factories/login', 'common/js/spec_helpers/ajax_helpers', 'common/js/components/utils/view_utils'],
+function($, LoginFactory, AjaxHelpers, ViewUtils) {
+    'use strict';
+    describe("Studio Login Page", function() {
+        var submit_button;
+
+        beforeEach(function() {
+            loadFixtures('login.underscore');
+            LoginFactory("/home/");
+            submit_button = $('#submit');
+        });
+
+        it('It will disable the submit button once its clicked', function() {
+            spyOn(ViewUtils, 'redirect').andCallFake(function(){});
+            var requests = AjaxHelpers.requests(this);
+            expect(submit_button).not.toHaveClass('is-disabled');
+            submit_button.click();
+            AjaxHelpers.respondWithJson(requests, {'success': true});
+            expect(submit_button).toHaveClass('is-disabled');
+        });
+
+        it('It will not disable the submit button if there are errors in ajax request', function() {
+            var requests = AjaxHelpers.requests(this);
+            expect(submit_button).not.toHaveClass('is-disabled');
+            submit_button.click();
+            AjaxHelpers.respondWithError(requests, {});
+            expect(submit_button).not.toHaveClass('is-disabled');
```

Comment:
Right. Right now, your test tests:
1) the submit button doesn't have the `is-disabled` class
2) the button is clicked
3) an ajax error response happens
4) the button doesn't have the `is-disabled` class

We're missing a step between 2 and 3, where we verify that while the ajax call is happening, the button is disabled (i.e., has the `is-disabled` class).